app/views/devise/sessions/bounced.html.slim

```diff
@@ -0,0 +1,8 @@
+- title t('headings.sp_redirect_bounced')
+
+h1.h3.my0 = t('headings.sp_redirect_bounced')
+p.mt2.mb2 == t('instructions.sp_redirect_bounced', application: @sp_name,
+        link: @sp_link.blank? ? @sp_name : link_to(@sp_name, @sp_link))
```

Comment:
What happens if you visit this link without an SP? We may want to put a conditional around everywhere we mention the SP.